nibabel/tests/test_scripts.py

```diff
@@ -67,6 +68,98 @@ def check_nib_ls_example4d(opts=[], hdrs_str="", other_str=""):
+                 "\n" + "slice_end  23                                           0                                     "
+                        "       " + "\n" + "xyzt_units 10                                           0                  "
+                                           "                          " + "\n" + "cal_max    1162.0                    "
+                                                                                 "                   0.0               "
+                                                                                 "                           " + "\n" +
+                 "descrip    b'FSL3.3\\x00 v2.25 NIfTI-1 Single file format'b''                                         "
+                 " \n" + "qform_code 1                                            0                                    "
+                         "        " + "\n" + "sform_code 1                                            2                "
+                                             "                            \n" +"quatern_b  -1.9451068140294884e-26    "
+                                                                               "                  0.0                  "
+                                                                               "                        " + "\n" +
+                 "quatern_c  -0.9967085123062134                          0.0                                          "
+                 + "\n" + "quatern_d  -0.0810687392950058                          0.0                                 "
+                          "         " + "\n" + "qoffset_x  117.8551025390625                            0.0            "
+                                               "                              " + "\n" + "qoffset_y  -35.72294235229492"
+                                                                                         "                           0."
+                                                                                         "0                            "
+                                                                                         "              " + "\n" +
+                 "qoffset_z  -7.248798370361328                           0.0                                          "
+                 + "\n" + "srow_x     [-2.0, 6.7147157e-19, 9.0810245e-18, 117.8551][1.0, 0.0, 0.0, 0.0]               "
+                          "          " + "\n" + "srow_y     [-6.7147157e-19, 1.9737115, -0.35552824, -35.722942][0.0, 3"
+                                                ".0, 0.0, 0.0]                         " + "\n" +
+                 "srow_z     [8.2554809e-18, 0.32320762, 2.1710818, -7.2487984][0.0, 0.0, 2.0, 0.0]                    "
+                 "     " + "\n" + "DATA: These files are different.\nChecksum   b0abbc49                    "
+                 + "                 0a2576dd")
```

Comment:
Could you clean up the white space on this? Also, string literals can be placed next to each other without `+`.